src/python/pants/backend/experimental/java/register.py

```diff
@@ -18,8 +18,9 @@
 from pants.jvm import util_rules as jvm_util_rules
 from pants.jvm.dependency_inference import symbol_mapper
 from pants.jvm.goals import coursier
+from pants.jvm.package.war import rules as war_rules
```

Comment:
`mypy` fails on this import even though the import works at runtime:

```
src/python/pants/backend/experimental/java/register.py:21:1: error: Cannot find
implementation or library stub for module named "pants.jvm.package.war"
[import]
    from pants.jvm.package.war import rules as war_rules
    ^
src/python/pants/backend/experimental/java/register.py:21:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 37 source files)
```

cc @Eric-Arellano  -- thoughts?

Reply:
Make sure you have `__init__.py` in every directory

Reply:
There are: 

```
$ find src/python/pants/jvm -name __init__.py
src/python/pants/jvm/test/__init__.py
src/python/pants/jvm/resolve/__init__.py
src/python/pants/jvm/goals/__init__.py
src/python/pants/jvm/__init__.py
src/python/pants/jvm/package/__init__.py
```

Reply:
Ah, BUILD file is empty

Reply:
doh!